src/config/config.go

```diff
@@ -41,38 +41,61 @@ func LoadConfig(filename string) (*Configuration, error) {
 	return &config, nil
 }
 
+// checks if each value in the config file is valid input
 func ValidateConfiguration(config *Configuration) error {
 
-	//check that the fileInputOption is either live or static
 	//convert all strings to lower case to ignore any capitalizations
 	fileInputL := strings.ToLower(config.FileInputOption)
-	outlierL := strings.ToLower(config.OutlierDetectionAlgorithm)
 
-	//added to check if no window size was put in
-	if config.WindowSize == "" {
-		config.WindowSize = "360"
-		fmt.Println("No window size was passed in. The default window size was set to 360")
+	//check that the fileInputOption is either live or static
+	if fileInputL != "live" && fileInputL != "static" {
+		return errors.New("DataOption in config.json must be either 'live' or 'static'")
 	}
 
+	//checks for valid input corresponding to choice of live vs static data analysis
 	if fileInputL == "live" {
```

Comment:
Nit pick - just to put this on your radar. Go supports switch case statements for future work.

Reply:
actually on second thought. We should do this. 

That way you dont need so many nested if statements.